tox.ini

```diff
@@ -1,8 +1,9 @@
 [tox]
 envlist =
-    py{36, 39}-{django21, django31}
-    py{38, 39}-{djangomain}
-
+    py36-django{21, 22, 30, 31, 32}
```

Comment:
What the reason behind adding `22`, `30` and `31` versions? We agreed to reduce the test time by using the oldest and most recent version along with main. The same applies for python versions.